packages/interactivity/src/directives.tsx

```diff
@@ -18,7 +18,14 @@ import {
 	splitTask,
 	isPlainObject,
 } from './utils';
-import { directive, getEvaluate, type DirectiveEntry } from './hooks';
+import {
+	directive,
+	getEvaluate,
+	isDefaultDirectiveSuffix,
+	isNonDefaultDirectiveSuffix,
```

Comment:
They're both type guards so `Array.prototype.filter` and `Array.prototype.find` work with them to narrow types. Type guards are one-sided, however, the `true` case is narrowed, so because we narrow to the default suffix and to a non-default suffix, it's necessary to have both.

This could be done other ways, but this seemed expressive and clear. I'd also originally implemented this with a `symbol` as the default value, but that wasn't necessary and switching the implementation was simple.